usaspending_api/api_contracts/contracts/v2/disaster/award/loans.md

```diff
@@ -0,0 +1,153 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Awards Spending Disaster/Emergency Funding via Loans [/api/v2/disaster/award/loans/]
+
+This endpoint provides insights on the Agencies awarding loans from disaster/emergency funding per the requested filters.
```

Comment:
`Agencies awarding loans from disaster/emergency funding` --> `awards which received disaster/emergency funding in the form of loans`?

Reply:
I made adjustments, see if they are good enough for now